sdks/go/examples/xlang/wordcount/xlang_wordcount.go

```diff
@@ -0,0 +1,100 @@
+package main
```

Comment:
You need to add Apache license to all new files.

Reply:
Ack.